src/Run/CommandLine.cs

```diff
@@ -762,8 +762,7 @@ public string GetHelp(int maxLineWidth, string parameterSetName = null, bool dis
                     }
                 }
             }
-
-            if (hasQualifiers)
+            else if (hasQualifiers)
```

Comment:
Looking at [this](https://github.com/dotnet/buildtools/blob/master/src/Run/CommandLine.cs#L736), they are mutually exclusive options.

Positional parameters are not being used by run.exe so this code is never going to be called. If there is a desire to start using them the design of run.exe should change and it would involve taking a look at the whole CommandLine class.
For now I consider it safe to leave it there as it is.